streaming/src/main/scala/org/apache/gearpump/streaming/transaction/api/TimeReplayableSource.scala

```diff
@@ -40,24 +41,16 @@ import org.apache.gearpump.{Message, TimeStamp}
  *     }
  *
  *     override def onNext(msg: Message): Unit = {
- *       TimeReplayableSource.pull(num).foreach { msg =>
+ *       TimeReplayableSource.read(num).foreach { msg =>
  *         TimeStampFilter.filter(msg, this.startTime).map(output)
  *       }
  *     }
  *   }
  *
  */
-trait TimeReplayableSource extends java.io.Serializable {
+trait TimeReplayableSource extends DataSource {
```

Comment:
Better document, hard to understand these 2 functions. 

If time starts from 0, and if I setStartTime(10) and then call startFromBeginning(), what data will I read, from 0 or from 10?